src/streams/StreamItem.js

```diff
@@ -151,13 +160,23 @@ export default function StreamItem(props: Props): Node {
       }}
     >
       <View style={wrapperStyle}>
+        {isCollapsible === true && (
+          <Touchable onPress={handleCollapse && (() => handleCollapse(streamId))}>
```

Comment:
One thing `Touchable` does is that when you touch it, it gives visual feedback filling the whole rectangle. (You can see this in the screencap.)

That's appropriate for something styled like a button, but it looks odd here. Instead, we can use `Pressable`. See `LightboxHeader` for a simple example, which makes the "X" icon at the top right of the lightbox screen.